phpunit.xml.dist

```diff
@@ -64,20 +64,20 @@
         -->
         <env name="TESTS_ZEND_LDAP_ONLINE_ENABLED" value="false" />
         <env name="TESTS_ZEND_LDAP_HOST" value="localhost" />
-        <env name="TESTS_ZEND_LDAP_PORT" value="389" />
-        <env name="TESTS_ZEND_LDAP_USE_START_TLS" value="true" />
+        <env name="TESTS_ZEND_LDAP_PORT" value="3890" />
+        <env name="TESTS_ZEND_LDAP_USE_START_TLS" value="false" />
         <env name="TESTS_ZEND_LDAP_USE_SSL" value="false" />
-        <env name="TESTS_ZEND_LDAP_USERNAME" value="CN=someUser,DC=example,DC=com" />
-        <env name="TESTS_ZEND_LDAP_PRINCIPAL_NAME" value="someUser@example.com" />
-        <env name="TESTS_ZEND_LDAP_PASSWORD" value="" />
+        <env name="TESTS_ZEND_LDAP_USERNAME" value="cn=Manager,dc=example,dc=com" />
+        <env name="TESTS_ZEND_LDAP_PASSWORD" value="insecure" />
         <env name="TESTS_ZEND_LDAP_BIND_REQUIRES_DN" value="true" />
-        <env name="TESTS_ZEND_LDAP_BASE_DN" value="OU=Sales,DC=example,DC=com" />
-        <env name="TESTS_ZEND_LDAP_ACCOUNT_FILTER_FORMAT" value="(&amp;(objectClass=posixAccount)(uid=%s))" />
+        <env name="TESTS_ZEND_LDAP_BASE_DN" value="dc=example,dc=com" />
```

Comment:
I've checked out your branch and did the following:

    vagrant up
    export TESTS_ZEND_LDAP_ONLINE_ENABLED=true
    phpunit

A lot of the tests that are skipped without the ```TESTS_ZEND_LDAP_ONLINE_ENABLED=true``` are now failing. Perhaps I did something wrong. I'll dig into it tonight (tight schedule today....)